src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreMeterProviderBuilderExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace OpenTelemetry.Metrics;
 /// <summary>
 /// Extension methods to simplify registering of ASP.NET Core request instrumentation.
 /// </summary>
-public static class MeterProviderBuilderExtensions
+public static class AspNetCoreMeterProviderBuilderExtensions
```

Comment:
Maybe we should go a little more specific like `AspNetCoreInstrumentationMeterProviderBuilderExtensions`?

Reply:
Sounds good to me. Done.